disdrodb/L0/readers/NCAR/parser_VORTE2_2010.py

```diff
@@ -0,0 +1,300 @@
+        a = a.reshape(int(len(a)/97),97)
+        df = dd.DataFrame(a)
+
+        # Remove number before data
+        for col in df:
+            df[col] = df[col].str[3:]
+
+        # Rename columns
+        import numpy as np
+        df.columns = np.arange(1,98)
+
+        col = {1: 'rainfall_rate_32bit',
+            2: 'rainfall_accumulated_32bit',
+            3: 'weather_code_synop_4680',
+            4: 'weather_code_synop_4677',
+            5: 'weather_code_metar_4678',
+            6: 'weather_code_nws',
+            7: 'reflectivity_32bit',
+            8: 'mor_visibility',
+            9: 'sample_interval',
+            10: 'laser_amplitude',
+            11: 'number_particles',
+            12: 'sensor_temperature',
+            13: 'sensor_serial_number',
+            14: 'firmware_iop',
+            15: 'firmware_dsp',
+            16: 'sensor_heating_current',
+            17: 'sensor_battery_voltage',
+            18: 'sensor_status',
+            19: 'start_time',
+            20: 'sensor_time',
+            21: 'sensor_date',
+            22: 'station_name',
+            23: 'station_number',
+            24: 'rainfall_amount_absolute_32bit',
+            25: 'error_code',
+            # 26: 'sensor_temperature_pcb',
+            # 28: 'sensor_temperature_receiver',
+            # 29: 'sensor_temperature_trasmitter',
+            30: 'rainfall_rate_16_bit_30',
+            31: 'rainfall_rate_16_bit_1200',
+            32: 'rainfall_accumulated_16bit',
+            # 33: 'reflectivity_16bit',
+            # 34: 'rain_kinetic_energy',
+            # 35: 'snowfall_rate',
+            # 60: 'number_particles_all',
+            # 61: 'list_particles',
+            90: 'raw_drop_concentration',
+            91: 'raw_drop_average_velocity',
+            92: 'raw_drop_number'}
+
+        df = df.rename(col, axis=1)
+
+        # Delete invalid columns by check_L0A
+        col_to_drop = ['rainfall_rate_16_bit_1200', 'rainfall_rate_16_bit_30', 'sample_interval', 93,94,95,96,97]
+        for c in col_to_drop:
+            if c not in df.columns:
+                col_to_drop.remove(c)
+        df = df.drop(columns=col_to_drop)
+
+        # Cast time
+        df["time"] = dd.to_datetime(df["sensor_date"] + '-' + df["sensor_time"], format="%d.%m.%Y-%H:%M:%S")
+        df = df.drop(columns=['sensor_date','sensor_time'])
+
+        # Drop useless columns
+        df = df.replace("", np.nan)
+        df = df.dropna(how='all', axis=1)
+
+
+        # Trim weather_code_metar_4678 and weather_code_nws
+        if set(['weather_code_metar_4678','weather_code_nws']).issubset(df.columns):
+            df['weather_code_metar_4678'] = df['weather_code_metar_4678'].str.strip()
+            df['weather_code_nws'] = df['weather_code_nws'].str.strip()
+
+        # Cast values
+        col_list = ['mor_visibility', 'number_particles', 'sensor_temperature',]
+        for c in col_list:
+            df[c] = df[c].str.split('.').str[0].astype(float)
```

Comment:
These are the corresponding fields in the sample data. Why you split by '.'  and do this strange `str[0]` ? 
```
08:5000
11:00000
12:019
```

Reply:
I did because these field shouldn't be float, and in some raw are saved like these:
```
08:9999.00
11:0.00000
12:31.0000
```
So in the loop I save only the decimal part